scifight/models.py

```diff
@@ -163,6 +167,9 @@ def clean(self):
     class Meta:
         unique_together = ("fight", "action_num")
 
+    def __str__(self):
+        return 'FIGHT.{0} ACTION.{1}'.format(self.fight, self.action_num)
+
```

Comment:
Let's rename `action_num` into `stage_num` for consistency, corresponding migration should be reversible and pretty simple.